frontend/src/components/NotifyModal.js

```diff
@@ -20,20 +10,27 @@ function NotifyModal() {
       return (
        <div className="notify-modal">
             <button className="notify-btn" onClick={() => setModalIsOpen(true)}>Notify Me!</button>
-            <Modal 
+            <ReactModal 
                 isOpen={modalIsOpen} 
-                onRequestClose={() => setModalIsOpen(false)} 
-                style={customStyles}
+                onRequestClose={() => setModalIsOpen(false)}
+                className="Modal"
+                overlayClassName="Overlay"
+                closeTimeoutMS={200} 
             >
-                <button className="close-btn" onClick={() => setModalIsOpen(false)}>X</button>
+                <div className="position-absolute top-0 end-0">
+                    <button className="close-btn" onClick={() => setModalIsOpen(false)}>
+                        <img className="xIcon" src={xIcon} alt="close button"></img>
+                    </button>
```

Comment:
the spacing seems a bit off here - it might be an issue with the icon itself. Perhaps we can use some other random "X" icon in the meantime as long as it is centered? 
![image](https://user-images.githubusercontent.com/42781446/185066399-2c42ba48-816e-4417-98d9-e82f3dd54a8f.png)